src/components/slider/slider.stories.ts

```diff
@@ -158,8 +155,8 @@ export const rangeLabeledTicksEdgePositioningAtMin_TestOnly = (): string => html
   ></calcite-slider>
 `;
 
-export const Histogram = (): HTMLCalciteSliderElement => {
-  const slider = document.createElement("calcite-slider") as HTMLCalciteSliderElement;
+export const Histogram = (): any => {
```

Comment:
Why the change to type as `any` in these cases?

Reply:
We don't have access to the component types from the stories so it throws linting errors
![image](https://user-images.githubusercontent.com/10986395/204942558-9b50cc71-54f2-4a23-ad9a-3842dfd55da2.png)

I'm pretty sure I tried importing `components.d.ts` at one point, which fixed the linting errors but caused build errors. I can try to dig that up again when I have some time

Reply:
Yeah it was in #5482

Here was the error that popped up when importing `components.d.ts`:
https://github.com/Esri/calcite-components/actions/runs/3275079841/jobs/5389458993#step:6:50